test/Tester/project.json

```diff
@@ -10,7 +10,8 @@
     "WindowsAzure.Storage": "7.0.0",
     "xunit": "2.1.0",
     "xunit.runner.visualstudio": "2.1.0",
-    "Xunit.SkippableFact": "1.2.14"
+    "Xunit.SkippableFact": "1.2.14",
+    "System.Management.Automation.dll": "10.0.10586"
```

Comment:
Not for now. Will keep it there. We need revisit all the tests and make a test kit otherwise we will still keeping the tests duplicated but this time, spread across multiple projects. 

I think we should design a new testkit I'm just not sure this is the right time since we have tons of other things to do for the .net core move...